.circleci/config.yml

```diff
@@ -0,0 +1,30 @@
+version: 2
+jobs:
+  build:
+    docker:
+    - image: circleci/node:8.11.1
+    steps:
+    - checkout
+    - run:
+        name: Code Linting
+        command: |
+          yarn && yarn lint
+  client-test:
+    docker:
+    - image: circleci/node:8.11.1
+    steps:
+    - checkout
+    - run:
+        name: run client tests
+        command: |
+          cd client && yarn test
+  
+  server-test:
+    docker:
```

Comment:
you can create a common step as build and use that build everywhere, try to reduce the code duplication.

Reply:
Will get it done right away.